.github/workflows/openms_ci_matrix_full.yml

```diff
@@ -889,17 +910,34 @@ jobs:
             octokit.rest.git.createRef({
               owner: context.repo.owner,
               repo: 'de.openms.knime',
-              ref: ${{ steps.extract_branch.outputs.RUN_NAME }},
+              ref: ${{ env.RUN_NAME }},
               sha: latest_ref,
             });
 
-      #- name: update links to latest (documentation, installers)
-      #  if: inputs.mark_as_latest
-      #  run: #FIXME
-      
       #- name: merge branch back into develop
-      #  if: inputs.mark_as_latest
-      #  run: #FIXME
+      - name: Merge to Develop
+        if: inputs.mark_as_latest
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { repo, owner } = context.repo;
+            const result = await github.rest.pulls.create({
```

Comment:
Hmm I dont think you need a PR. release branches should ALWAYS be ff-only mergeable into main/master. If not, you should not even start the release ;) Maybe check this in the very beginning and maybe also tag the merged commit instead of the head commit of the release branch (although it might be that the head commit carries over its tags when you merge it ff-only into master/main).
What would be nice, is to have a PR for master or release to be merged back into develop, though!